src/mnemonic-key.spec.ts

```diff
@@ -41,4 +47,19 @@ describe('Mnemonic Key Class', () => {
     expect(instance.signer).to.exist
     expect(instance.signer?.privateKey).to.exist
   })
+
+  it('should throw an error if mnemonic key is not valid', () => {
+    try {
+      new MnemonicKey(
+        HARMONY_RPC_SHARD_0_TESTNET.url,
+        {
+          mnemonic: 'this is an wrong example of seed',
+        },
+        ChainID.HmyTestnet,
+      )
+    } catch (error) {
+      expect(error).to.be.exist
+      expect(error).to.be.instanceOf(Error)
+    }
+  })
```

Comment:
maybe we can also add a test that checks that after creating a `MnemonicKey` instance with `numberOfAccounts` > 1, that there are in fact `n` accounts in the `accounts` property.

So for example:
```
const key = new MnemonicKey(
        HARMONY_RPC_SHARD_0_TESTNET.url,
        {
          mnemonic: '...',
          numberOfAddresses: 5
        },
        ChainID.HmyTestnet,
      )
      
// Here: Check that key.accounts has 5 accounts
```

Reply:
Mmmmh, okay! Let's do that!